src/popup/snooze-content.js

```diff
@@ -66,6 +72,7 @@ function openSnoozedTab(item) {
     active: true,
     url: item.url
   });
+  cancelSnoozedTab(item);
```

Comment:
Hmm, that's rather unrelated now that I think about it. I think I slipped it in as something I thought was missing - i.e. a snoozed tab waking up should probably get removed from the list. But, I can pull that out and handle it in another PR.